test/JDBC/expected/database_metadata.out

```diff
@@ -0,0 +1,276 @@
+
+create table dbmeta_tab1(col1 int primary key, col2 nvarchar(max) not null)
```

Comment:
We should create a view as well to see APIs like getTables() show views as well

Reply:
Added a view.